Tools/build/deepfreeze.py

```diff
@@ -297,10 +297,12 @@ def generate_code(self, name: str, code: types.CodeType) -> str:
             self.write(f".co_linetable = {co_linetable},")
             self.write(f"._co_cached = NULL,")
             self.write(f".co_code_adaptive = {co_code_adaptive},")
-            for i, op in enumerate(code.co_code[::2]):
+            first_traceable = 0
+            for op in code.co_code[::2]:
                 if op == RESUME:
-                    self.write(f"._co_firsttraceable = {i},")
                     break
+                first_traceable += 1
+            self.write(f"._co_firsttraceable = {first_traceable},")
```

Comment:
Why did you have to rewrite this? It would seem to result in the same outcome? I.e. if co_code starts with
```
  XX
  YY
  ZZ
  RESUME
```
it should set `co_firsttraceable` to 3.

Reply:
The outcome is different if you don't have RESUME in the code. This shouldn't happened, but I was debugging a bug where this code thought that it did happen. This calculation is [also implemented in codeobject.c](https://github.com/python/cpython/blob/e88eb3775ecdcb3af6c6d694a935b7fa5f41e5ce/Objects/codeobject.c#L441), and in that case it gave a different number in the two places.  So this change it to make the two versions of the calculation do the same thing.